### PR TITLE
[SCL-45] function for completing a task done and tested

### PR DESCRIPTION
**Pull request by JessieChiu1**
- concatenated the functions for completing a task (Api.tsx)
- tested functions on App.tsx and firebase real time DB updated

Addresses SCL-45

### CODE REVIEW

**Review by jacoblb64 — CHANGES_REQUESTED** [review]
Looks good overall! A few small changes I'd suggest to get this over the line. Blocking for me is restoring the `.env_template` file.

**Comment by jacoblb64** [comment]
Unrelated, but I realized I forgot to tell you that you can copy the branch name from the Linear ticket -- it'll autolink the issue!
https://linear.app/changelog/2020-04-13-branch-naming

File: app/.env_template
[the remark this answers is not in the document]
Comment (JessieChiu1) [reply app/.env_template:1]: Yes, I followed the setup instruction and must have removed it.

I changed the .env file back to .env_template.

File: app/src/App.tsx [hunk 1]
```diff
@@ -147,52 +147,55 @@ function App() {
...
-      const addRaffleTicketEntry = async () => {
-        const raffleId = 'some_raffle_id';
-        const numberOfEntries = 1;
-        await firebaseService.addRaffleEntry(user.uid, raffleId, numberOfEntries);
-      };
+      // Jess - CompleteTask testing
+      /**
+       * generate a task
+       * run the firebaseService class function (completeTask) from Apt.tsx
+       * (Please check to make sure it is adding 1 entry and 5 tickets_remaining)
```
[the remark this answers is not in the document]
Comment (JessieChiu1) [reply app/src/App.tsx]: It is Stanley's testing function, I just added the comment to make sure I am understanding what is going on. I can remove them if you need me to.

File: app/src/Api.tsx [hunk 2]
```diff
@@ -70,53 +70,50 @@ export class FirebaseService {
...
-        return (currentTickets || 0) - decrement;
+        return (currentTickets || 0) + increment;
       });
-      console.log("tickets_remaining decremented successfully.");
+      console.log("tickets_remaining incremented successfully.");
     } catch (error) {
-      console.error("Error decrementing tickets_remaining:", error);
+      console.error("Error completing task:", error);
     }
   }
```
[the remark this answers is not in the document]
Comment (JessieChiu1) [reply app/src/Api.tsx]: - I removed the increment. I wasn't sure if there were tasks that rewards more tickets.
- added the taskId to the error log
- I can add the borough enum, but I might need more instruction.

File: app/src/App.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jacoblb64) [reply app/src/App.tsx]: Ah no worries, thanks for the explanation, just wasn't sure if I was missing something

File: app/src/Api.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jacoblb64) [reply app/src/Api.tsx]: Looks good, thank you! We can do some borough refactoring later for safety, don't worry about it now

**Review by jacoblb64 — APPROVED** [review]
Thanks for making the requested change, this is good to merge!

**Pull request merged**